app/managers.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self._table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self._table_name}"
+            f" (first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list:
+        cursor = self._connection.execute(
+            f"SELECT * FROM {self._table_name}"
+        )
+        return [
+            row for row in cursor
+        ]
+
+    def delete(self, id_: int) -> None:
+        self._connection.execute(
+            f"DELETE FROM {self._table_name} WHERE id = ?",
+            (id_,)
+        )
+
+        self._connection.commit()
+
+    def update(self, id_: int, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"UPDATE {self._table_name} "
```

Comment:
Using f-strings for SQL queries can lead to SQL injection vulnerabilities. Consider using parameterized queries instead.